coprocessor/fhevm-engine/scheduler/src/dfg/scheduler.rs

```diff
@@ -764,58 +385,270 @@ fn partition_components(
     Ok(())
 }
 
-type TaskResult = Result<(SupportedFheCiphertexts, Option<(i16, Vec<u8>)>)>;
+fn re_randomise_transaction_inputs(
+    inputs: &mut HashMap<Handle, Option<DFGTxInput>>,
+    transaction_id: &Handle,
+    gpu_idx: usize,
+    cpk: tfhe::CompactPublicKey,
+) -> Result<()> {
+    let transaction_rerandomisation_domain_separator = *b"TFHE-w.r";
+    let compact_public_encryption_domain_separator = *b"TFHE.Enc";
+    let mut re_rand_context = ReRandomizationContext::new(
+        transaction_rerandomisation_domain_separator,
+        // First is the function description, second is a nonce
```

Comment:
comment looks copy pasted without nonce, though transaction ID acts as a function description and nonce itself I guess ?